.gitlab-ci.yml

```diff
@@ -94,6 +94,15 @@ test_macos11_py27_coremltools_test:
   variables:
     WHEEL_PATH: build/dist/*cp27*10_16*
     PYTHON: "2.7"
+  only:
+    changes:
+      - coremltools/test/**/*.{py}
```

Comment:
Should this also trigger on changes to .proto? Or .cpp?

Reply:
validator changes should run all the tests (at least as of now, until we have something based on op related filtering)

Reply:
Fixed.